lib/src/component/abstract_transition.dart

```diff
@@ -368,6 +369,23 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps,
     }
   }
 
+  /// Returns attributes only available during testing that indicate the state of the transition.
+  Map<String, String> getTransitionTestAttributes() {
+    if (!component_base.UiProps.testMode) return const {};
+
+    const enumToAttrValue = const <TransitionPhase, String>{
```

Comment:
Noobie question - what is the value of "TransitionPhase"? I assume this is the attribute I'll have to search for in my functional tests when checking if the modal dialog has finished animating.

Reply:
@seangerhardt-wf the HTML will look like this based on the current transition phase _(only listing the ones you'd probably be interested in for functional test purposes)_:

`TransitionPhase.SHOWN`

```html
<div data-transition-phase="shown"><!-- ... -->
```

`TransitionPhase.HIDDEN`

```html
<div data-transition-phase="hidden"><!-- ... -->
```